src/components/SitewideHeader/index.js

```diff
@@ -29,6 +29,18 @@ const SitewideHeader = ({
 
           <nav className={classes.fullScreenNav} aria-label="Sitewide navigation">
             <span>
+              {['lg', 'xl', 'none'].includes(maxWidthBreakpoint) && (
```

Comment:
Without this, there are too many links in the header on small screen sizes which are not small enough to be mobile width.